src/main.ts

```diff
@@ -1 +1,31 @@
-console.log('main process');
+const DATA_FETCH_FREQUENCY = 5000;
+const BEACON_UPDATE_FREQUENCY = 10000;
```

Comment:
Nit: 
```suggestion
const BEACON_UPDATE_FREQUENCY = 10_000;
```

I also like appending the unit to the variable name (i.e. `_MS`) but definitely not super important